WORKSPACE

```diff
@@ -81,3 +81,9 @@ load(
 )
 
 _go_image_repos()
+
+git_repository(
```

Comment:
This should not go here.  `diff.bzl` should define `repositories` function that imports Go (which this needs) and this repo.  You should use the same logic as in the other `repositories` functions to avoid reimporting an existing copy of either.